setup.py

```diff
@@ -0,0 +1,18 @@
+from setuptools import setup
```

Comment:
Do we think this setup.py will eventually grow to install other tools? Or should this be segregated into an oep2 directory?